<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDIMISSION</class>
 <widget class="QWidget" name="CDIMISSION">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>523</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="Dimission_stacked">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>767</width>
     <height>523</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QTableWidget" name="QDimission">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>85</y>
       <width>701</width>
       <height>380</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="QpbtnSearch">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>43</y>
       <width>36</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="shortcut">
      <string>Return</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Keyword_Edit">
     <property name="geometry">
      <rect>
       <x>36</x>
       <y>43</y>
       <width>257</width>
       <height>36</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="QDimissionList">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>85</y>
       <width>701</width>
       <height>380</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="QpbtnChange">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>切换</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QPushButton" name="QpbtnSave">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>440</y>
       <width>96</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Head_Photo_D">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>45</y>
       <width>91</width>
       <height>96</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="QpbtnClose">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>440</y>
       <width>96</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>285</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>处理时间：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Number_D">
     <property name="geometry">
      <rect>
       <x>216</x>
       <y>75</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Job_D">
     <property name="geometry">
      <rect>
       <x>216</x>
       <y>165</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Name_D">
     <property name="geometry">
      <rect>
       <x>216</x>
       <y>45</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="MakeMan_D">
     <property name="geometry">
      <rect>
       <x>228</x>
       <y>315</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>105</y>
       <width>36</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>性别：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>75</y>
       <width>36</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>工号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>135</y>
       <width>36</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>部门：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>165</y>
       <width>36</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>职位：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>195</y>
       <width>72</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>身份证号码：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>45</y>
       <width>36</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>姓名：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>225</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>入职时间：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>255</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>离职时间：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>315</y>
       <width>48</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>处理人：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>345</y>
       <width>36</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>原因：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Reason_D">
     <property name="geometry">
      <rect>
       <x>216</x>
       <y>345</y>
       <width>250</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ID_Number_D">
     <property name="geometry">
      <rect>
       <x>252</x>
       <y>195</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="QpbtnRetuen">
     <property name="geometry">
      <rect>
       <x>703</x>
       <y>331</y>
       <width>64</width>
       <height>64</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="QpbtnLogout">
     <property name="geometry">
      <rect>
       <x>703</x>
       <y>395</y>
       <width>64</width>
       <height>64</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="QpbtnEdit">
     <property name="geometry">
      <rect>
       <x>703</x>
       <y>459</y>
       <width>64</width>
       <height>64</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="Sex_D">
     <property name="geometry">
      <rect>
       <x>216</x>
       <y>105</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>男</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>女</string>
      </property>
     </item>
    </widget>
    <widget class="QDateEdit" name="Entry_Time_D">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>225</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="Dimission_Time_D">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>255</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="Leave_Deal_Day_D">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>285</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="Department_D">
     <property name="geometry">
      <rect>
       <x>216</x>
       <y>135</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
